contracts/interchain-token-service/src/state.rs

```diff
@@ -294,7 +294,10 @@ pub fn save_token_config(
 pub fn save_custom_token(
     storage: &mut dyn Storage,
     chain: ChainNameRaw,
-    RegisterTokenMetadata { address, decimals }: RegisterTokenMetadata,
+    RegisterTokenMetadata {
+        token_address: address,
```

Comment:
```suggestion
        token_address,
```
use `token_address` as the variable below